cornell-dti/cu-apts
# Homepage listings with mobile responsiveness

## Summary

- Closes issue #14 
- Changes from pull request #18
- This pull request is for the implementation of the homepage design that includes apartment listings, and supports mobile responsiveness.
### Before/After

- The code has been largely refactored to support mobile responsiveness, and is now using the column/grid components provided by the `react-bootstrap` library.
- The design of the listing cards has been changed to match the current design on Figma.
- Currently, the listing cards are populated with hardcoded data, but they have been made functional in a way that populating them with backend data will not be difficult.
### Testing

- Run `yarn install` in `root` to make sure all packages are installed.
- Run `yarn start` in `root` to view the React project in a local browser. It will open to the Home page, where you can see the implementation of the new design.
- Right click the page and select `Inspect` to see how the webpage looks on different sized screens.
